packages/app/src/components/home/HomePage.tsx

```diff
@@ -82,7 +83,9 @@ const CatalogCards = () => {
   const [{ loading, error }, refresh] = useAsyncFn(
     async () => {
       const response = await catalogApi.getEntities();
-      setEntities(response.items.filter(e => e.kind === 'Component'));
+      setEntities(response.items.filter(
+        e => e.kind === 'Component' && e.metadata.annotations?.[FEATURED_ANNOTATION] === 'true'
```

Comment:
I think this query should be a tiny bit more clever...

1. With the new entity mapping per ADR, we should not scope to `Component` kind only.
2. Maybe we should probably require `logo-url` annotation to be present on the entity. WDYT?

Reply:
Yea that sounds good, I'll make sure to also update docs.

Reply:
Resolved in the latest commits.